client/http/http_sender_test.go

```diff
@@ -15,19 +15,22 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package client
+package http
```

Comment:
Might be better to name this package `httpsender` and the `client/rpc` package `client/rpcsender`. That would avoid the clashes with the `http` and `rpc` packages.